tools/Makefile.am

```diff
@@ -4,10 +4,10 @@ bin_PROGRAMS = lvm-cache-stats vfat-resize
 lvm_cache_stats_CFLAGS   = $(GLIB_CFLAGS) $(BYTESIZE_CFLAGS) -Wall -Wextra -Werror
 lvm_cache_stats_CPPFLAGS = -I${builddir}/../include/
 lvm_cache_stats_LDFLAGS  = -Wl,--no-undefined
-lvm_cache_stats_LDADD    = ${builddir}/../src/lib/libblockdev.la $(GLIB_LIBS) $(BYTESIZE_LIBS)
+lvm_cache_stats_LDADD    = ${builddir}/../src/lib/libblockdev.la $(GLIB_LIBS) $(BYTESIZE_LIBS) $(UUID_LIBS)
```

Comment:
Hmm, is this a separate linking issue? I don't see any mention of libparted or uuid in `lvm_cache_stats`.

Reply:
You're right. Updated. Sorry for late reaction!